lib/__tests__/dom.nested.test.js

```diff
@@ -51,7 +51,9 @@ describe('DocumentMeta - DOM nested', () => {
       <div>
         <DocumentMeta {...DOC_META} />
         <div>
-          <DocumentMeta {...DOC_META_NESTED} extend />
+          <DocumentMeta {...DOC_META_NESTED} extend>
+            <div>Hello world!</div>
```

Comment:
What is the reason, for nesting the DOM elements in the first place? DocumentMeta is NOT supposed to have nested DOM elements.

Why don't you just create it as a sibling? DocumentMeta does not render anything to your DOM.